packages/devextreme/js/__internal/ui/splitter/splitter_item.ts

```diff
@@ -59,6 +59,13 @@ class SplitterItem extends CollectionWidgetItem {
   isLast(): boolean {
     return this._owner._isLastVisibleItem(this.getIndex());
   }
+
+  _dispose(): void {
+    // @ts-expect-error badly typed base class
```

Comment:
We shouldn't leave ignore comments.

Reply:
we have more than 10 ts-expect-error statemenst left in our splitter component, i suggest fixing them separately. I dont think this one have a quick fix coz it looks like it should be added somewhere on a base level